src/components/tech/HeroSection.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import { Box, Flex, Heading, Text, Button } from "@radix-ui/themes";
+import heroBackgroundImg from "../../../public/images/tech/tech-hero-background-img.jpg";
+const HeroSection = () => {
+  return (
+    <div className="block w-[1252px] item-center justify-center p-8 relative ">
+      <h1 className="text-center text-white uppercase font-bold md:text-5xl md:py-6 py-2 z-20 sm:text-3xl text-xl">
+        TECH AT DISTRIBUTE AID
+      </h1>
+
+      <Box className="w-full relative flex align-center items-center justify-center mt-4">
+        <Image
+          src={heroBackgroundImg}
+          alt="Hero Background"
+          className="w-full h-[520px] object-cover rounded-bl-[32px] rounded-br-[32px]"
```

Comment:
**issue (blocking):** This approach is causing style issues on some mobile browsers.  Instead of setting a specific-height image with the content absolutely positioned on top, could we instead set a background image on the content?